guides/tutorials/agent-config.md

```diff
@@ -190,6 +194,12 @@ a JSON object like the
 
 Stringified JSON object of the transaction.
 
+### `indyLedgers.transactionAuthorAgreement`
+
+**Type**: `TransactionAuthorAgreement`
+
+JSON representation specifying the version and acceptance mechanism.
```

Comment:
Could you explain what the fields inside this object mean? In the same structure as the other fields